pkg/disk/disk.go

```diff
@@ -148,28 +145,16 @@ func GlobalConfigSet(m metadata.MetadataProvider) utils.Config {
 	configMapName := "csi-plugin"
 
 	// Global Configs Set
-	cfg, err := clientcmd.BuildConfigFromFlags(options.MasterURL, options.Kubeconfig)
+	cfg, err := options.GetRestConfig()
```

Comment:
all the config is generated by func `GetRestConfig` in pkg/options/genericoptions.go, including content type setting. @mowangdk